Polyhedron/doc/Polyhedron/dependencies

```diff
@@ -7,3 +7,4 @@ Stream_support
 HalfedgeDS
 Miscellany
 BGL
+Polygon_mesh_processing
```

Comment:
Is it not a problematic new dependency?
(Cc: @afabri @sloriot)

Reply:
Sorry, forget my comment: I now see that it is only a dependency for the documentation.